index.js

```diff
@@ -26,8 +26,24 @@ maths.import({
 
 function transformResolve(value) {
     return reduceFunctionCall(value, 'resolve', function(argString) {
+
+        var unit = '';
+        if (argString.indexOf('floor(') >= 0
```

Comment:
Wow, wonder why they didn't want that feature! Seems quite deliberate like you say. Lets keep it simple then. We could replace the `if (argString.indexOf('floor...` part with sort of `while` loop to capture multiple floor/ceiling calls? Hows that sound?